api/src/main/java/org/openmrs/module/pihmalawi/metadata/deploy/bundle/concept/TraceConcepts.java

```diff
@@ -0,0 +1,33 @@
+package org.openmrs.module.pihmalawi.metadata.deploy.bundle.concept;
+
+
+import org.openmrs.Concept;
+import org.openmrs.api.ConceptNameType;
+import org.openmrs.module.metadatadeploy.builder.ConceptBuilder;
+import org.openmrs.module.metadatadeploy.bundle.Requires;
+import org.openmrs.module.pihmalawi.metadata.deploy.bundle.VersionedPihConceptBundle;
+import org.springframework.stereotype.Component;
+
+import java.util.Locale;
+
+@Component
+@Requires({CoreConceptMetadataBundle.class})
+public class TraceConcepts extends VersionedPihConceptBundle {
+
+
+    @Override
+    public int getVersion() {
+        return 1;
+    }
+
+    @Override
+    protected void installNewVersion() throws Exception {
+
+
+        install(new ConceptBuilder("C0730C1B-E6D3-43DD-A11B-99E9D22C5048")
```

Comment:
I think we need a new concept for "Date of Next Tracking Attempt" too, and not just re-purpose Next Visit Date, as I think this will get confusing.